gluten-core/src/main/scala/io/glutenproject/execution/WholeStageTransformer.scala

```diff
@@ -58,18 +65,14 @@ trait TransformSupport extends GlutenPlan {
    */
   def columnarInputRDDs: Seq[RDD[ColumnarBatch]]
 
-  def getBuildPlans: Seq[(SparkPlan, SparkPlan)]
-
-  def getStreamedLeafPlan: SparkPlan
```

Comment:
These two methods are from first initial commit, but it seems we never use them now. I'm not sure the original idea. cc @rui-mo @zhztheplayer @PHILO-HE  @zzcclp if you have some context

Reply:
It's some legacy code copied from Gazelle.

Reply:
Thanks. It looks good to me to remove them from the code.